loadgen/helpers.go

```diff
@@ -0,0 +1,34 @@
+package loadgen
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
+)
+
+func VisibilityCountIsEventually(
+	ctx context.Context,
+	client client.Client,
+	request *workflowservice.CountWorkflowExecutionsRequest,
+	expectedCount int,
+	waitAtMost time.Duration,
+) error {
+	deadline := time.Now().Add(waitAtMost)
+	for {
```

Comment:
Often more clearly implemented as a time.After + time.Ticker instead of a naive sleep so you don't have to wait the interval